### DISCUSSION TITLE
Shorter Info for Updater Option

### DISCUSSION THREAD

**Discussion by vedxp** [libre-tube/LibreTube#1634] [Ideas]
Currently, in the last option of settings; the update checker's info text is `Click to find out if the app is up to date.`

<details><summary>Show reference</summary>

![Screenshot_20221021-205834~2](https://user-images.githubusercontent.com/87353286/197326152-64d861a1-ffef-4eb2-9b88-baf0cfd20d9e.png)

</details>

There could be something like `Check for update` which is short but still expressive of what it'll do.

**Comment by vedxp** [comment]
@Bnyro have a look into it :)

**Reply by Bnyro** [reply]
Have a look at my Github profile status :)
